runtime/v1/shim/service.go

```diff
@@ -128,8 +128,8 @@ func (s *Service) Create(ctx context.Context, r *shimapi.CreateTaskRequest) (_ *
 	rootfs := ""
 	if len(mounts) > 0 {
 		rootfs = filepath.Join(r.Bundle, "rootfs")
-		if err := os.Mkdir(rootfs, 0711); err != nil && !os.IsExist(err) {
```

Comment:
sorry. I don't think it it good choice. if you don't want to keep burden to remember when I should `:=` or `=`, you can pick up different name like `retErr` or `err0`. Therefore, you can use it in defer without the burden.

Reply:
Naming for return error, retErr in this PR, is along the lines of what you suggested.

Reply:
Sorry for misunderstanding. `retErr` is the same to `err`, right? just different name. Could you tell me the truly purpose of your change? thanks

Reply:
ignore this comment. will comment your other pr.